base_layer/core/src/base_node/states/block_sync.rs

```diff
@@ -128,9 +167,9 @@ impl BlockSyncStrategy {
 }
 
 /// State management for BlockSync -> Listening.
-impl From<BlockSyncStrategy> for ListeningInfo {
+impl From<BlockSyncStrategy> for ListeningData {
```

Comment:
These won't be needed

Reply:
They are currently only placeholders for if we want data there